litho-it/src/test/java/com/facebook/litho/widget/TestSectionsRecyclerView.java

```diff
@@ -0,0 +1,68 @@
+package com.facebook.litho.widget;
+
+import android.content.Context;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.recyclerview.widget.RecyclerView;
+import java.util.ArrayList;
+import java.util.List;
+
+/** A test {@link SectionsRecyclerView} class used for unit testing. */
+public class TestSectionsRecyclerView extends SectionsRecyclerView {
+
+  private final List<Boolean> setRefreshingValues = new ArrayList<>();
+  private final List<Runnable> removeCallbackRunnableList = new ArrayList<>();
+  private final List<Runnable> postRunnableList = new ArrayList<>();
+  private OnRefreshListener listener;
+
+  public TestSectionsRecyclerView(@NonNull Context context, RecyclerView view) {
+    super(context, view);
+  }
+
+  public List<Boolean> getSetRefreshingValues() {
```

Comment:
Do we need all these actual collections or just `count` will be enough for each field?

Reply:
Right, I changed this to hold the count and hold the last refreshing value.